doc/iterations/2025.2/Improve_image_export.adoc

```diff
@@ -0,0 +1,40 @@
+= (S) Improve the image export
+
+== Problem
+
+Sirius Web is able to export image in the SVG format but, most of its content uses `<ForeignObject/>` to put HTML directly inside.
+Using `<ForeignObject/>` in the SVG prevent the SVG to be displayed outside a web browser.
+
+== Key Result
+
+An image exported from a Sirius Web diagram can import in a document that can read image like Word or Libre Office.
+
+=== Solution
+
+There are more than one solution to resolve that issue, we could find an existing solution, export the image in another format than SVG or implement a custom solution to keep the SVG format.
+
+==== Existing Solution
+
+We were not able to find any existing solutions that can fit our need:
+
+* We tried using https://github.com/vercel/satori:"vercel/satori" but it cannot work because it expects pure and stateless JSX, that our `<DiagramRender/>` is not.
+* We also tried https://www.npmjs.com/package/dom-to-svg:"dom-to-svg", which is also not working because, it relies on containment to compute children position and Reactflow put all nodes at the same depth in the DOM.
+
+==== Export in another format
+
+We are currently exporting a diagram in SVG using https://www.npmjs.com/package/html-to-image:"html-to-image".
+We could choose to export a diagram in another form using the same library.
+We tried to export the image in PNG, and we were rapidly able to export a quite big diagram with a good resolution.
+
+image::images/papaya_diagram.png[Quite big diagram]
```

Comment:
While it's a great example for the shape, we will not in anyway add a 17MB images to the Git repository, it would have a noticeable impact on performances later down the line, this should only be kept here as a temporary example which we won't merge.